tests/test_untrack_command.rs

```diff
@@ -80,8 +80,9 @@ fn test_untrack() {
     assert_eq!(
         stderr,
         format!(
-            "Error: '{}' and 1 other files would be added back because they're not ignored. Make \
-             sure they're ignored, then try again.\n",
+            "Error: '{}' and 1 other files are not ignored.\n\
+             Hint: Files that are not ignored will be added back by the next command.\n\
+             Make sure they're ignored, then try again.\n",
```

Comment:
Wrong commit?

Reply:
Indeed. I just noticed while fixing the formatting errors. Fixed.